webapp/channels/src/components/password_reset_form/password_reset_form.tsx

```diff
@@ -101,4 +99,4 @@ const PasswordResetForm = ({location, siteName, actions}: Props) => {
     );
 };
 
-export default memo(PasswordResetForm);
+export default memo(injectIntl(PasswordResetForm));
```

Comment:
Since this is a functional component you can directly use the hook https://formatjs.io/docs/react-intl/api/#useintl-hook